core/templates/dev/head/components/ExplorationCreationService.js

```diff
@@ -37,27 +35,25 @@ oppia.factory('ExplorationCreationService', [
 
         explorationCreationInProgress = true;
         alertsService.clearWarnings();
-        if (urlService.getPathname() !== '/my_explorations') {
-          $window.location.replace('/my_explorations?mode=create');
-        } else {
-          $rootScope.loadingMessage = 'Creating exploration';
-          $http.post('/contributehandler/create_new', {}, {
-            requestIsForCreateExploration: true
-          }).then(function(response) {
-            siteAnalyticsService.registerCreateNewExplorationEvent(
-              response.data.explorationId);
-            $timeout(function() {
-              $window.location = UrlInterpolationService.interpolateUrl(
-                CREATE_NEW_EXPLORATION_URL_TEMPLATE, {
-                  exploration_id: response.data.explorationId
-                }
-              );
-            }, 150);
-            return false;
-          }, function() {
-            $rootScope.loadingMessage = '';
-          });
-        }
+        $rootScope.loadingMessage = 'Creating exploration';
+
+        $http.post('/contributehandler/create_new', {}, {
+          requestIsForCreateExploration: true
+        }).then(function(response) {
+          siteAnalyticsService.registerCreateNewExplorationEvent(
```

Comment:
Are we also registering the create collection event in the CollectionCreationService?